src/m365/outlook/commands/room/room-list.ts

```diff
@@ -0,0 +1,66 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  roomlistEmail?: string;
+}
+
+class OutlookRoomListCommand extends GraphCommand {
```

Comment:
Since this is a list command we recommend using the GraphItemsListCommand base class. You can take the serviceannouncement-message-list command as reference

Reply:
Thank you. This was helpful. Implemented using GraphItemsListCommand